{"name": "test_delete_post", "status": "failed", "statusDetails": {"message": "AssertionError: \u001b[31mFAILURE: \u001b[0mОжидался статус код 404, но получен 200\nassert 200 == 404\n +  where 200 = <Response [200]>.status_code\n\u001b[31mtest_json_placeholder.py\u001b[0m:89 in test_delete_post() -> with check:\n\u001b[31mtest_json_placeholder.py\u001b[0m:90 in test_delete_post -> assert response.status_code == 404, f\"Ожидался статус код 404, но получен {response.status_code}\"\n\u001b[31mAssertionError: Ожидался статус код 404, но получен 200\nassert 200 == 404\n +  where 200 = <Response [200]>.status_code\n\u001b[0m\n\n------------------------------------------------------------\nFailed Checks: 1", "trace": "\u001b[31mFAILURE: \u001b[0mОжидался статус код 404, но получен 200\nassert 200 == 404\n +  where 200 = <Response [200]>.status_code\n\u001b[31mtest_json_placeholder.py\u001b[0m:89 in test_delete_post() -> with check:\n\u001b[31mtest_json_placeholder.py\u001b[0m:90 in test_delete_post -> assert response.status_code == 404, f\"Ожидался статус код 404, но получен {response.status_code}\"\n\u001b[31mAssertionError: Ожидался статус код 404, но получен 200\nassert 200 == 404\n +  where 200 = <Response [200]>.status_code\n\u001b[0m\n\n------------------------------------------------------------\nFailed Checks: 1"}, "start": 1736978074738, "stop": 1736978075311, "uuid": "4e742d16-e835-497d-9cba-9854f05c7901", "historyId": "8609e47d091832dae1ff3e8f3e85303f", "testCaseId": "8609e47d091832dae1ff3e8f3e85303f", "fullName": "tests.api_testing.test_json_placeholder#test_delete_post", "labels": [{"name": "parentSuite", "value": "tests.api_testing"}, {"name": "suite", "value": "test_json_placeholder"}, {"name": "host", "value": "DESKTOP-M99MFDK"}, {"name": "thread", "value": "15304-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testing.test_json_placeholder"}]}